### DISCUSSION TITLE
Redis Object Cache Bytes

### DISCUSSION THREAD

**Discussion by Dannyke77** [rhubarbgroup/.github#11] [Support (ROC)] [answered]
Hello i am using Redis object cache plugin for wordpress. And in the server memory is set to maximum 10gb.
But in wordpress i see the metric bytes not go higher then 5mb it shows its at 100% then. Also wp memory limit is set to 1024mb to see if that works but still same result. 

My server is a plesk / Centos 7 system

Any idea what i am doing wrong ?

Greets danny

Status: Connected
Client: PhpRedis (v5.3.7)
Drop-in: Valid
Disabled: No
Ping: 1
Errors: []
PhpRedis: 5.3.7
Relay: Not loaded
Predis: 2.1.2
Credis: Not loaded
PHP Version: 8.1.19
Plugin Version: 2.4.1
Redis Version: 7.0.11
Multisite: No
Metrics: Enabled
Metrics recorded: 335
Filesystem: Working

**Comment by tillkruss** [comment, marked as answer]
Hey Denny, you're doing nothing wrong. The "Bytes" refer to the number of bytes transmitted per page load. We may need a better name for them.
